src/css/_guest-layout.scss

```diff
@@ -142,3 +141,38 @@
   font-family: $secondary-font;
   margin-top: 10px;
 }
+
+.available-booking-card {
+  font-family: $secondary-font;
+  width: 95%;
+  padding: 5%;
+  border-bottom: black solid 1px;
+  display: flex;
+  flex-direction: row;
```

Comment:
I like your use of padding on the bottom boarder in order to make a "partition" between possible bookings.